doc/classes/BoneAttachment3D.xml

```diff
@@ -33,7 +39,14 @@
 				A function that is called automatically when the [Skeleton3D] is updated. This function is where the [BoneAttachment3D] node updates its position so it is correctly bound when it is [i]not[/i] set to override the bone pose.
 			</description>
 		</method>
-		<method name="set_external_skeleton">
+		<method name="set_external_skeleton" deprecated="Use [method set_external_skeleton_path] instead.">
```

Comment:
This is referenced below in `set_use_external_skeleton` and should be changed to point to the new method

Reply:
Good spot. Fixed